portfolio/src/main/java/com/google/sps/data/CheckLoginResponse.java

```diff
@@ -6,10 +6,12 @@
   private final Boolean loggedIn;
   private final String url; 
   private final String email;
+  private final Boolean access;
 
-  public CheckLoginResponse(Boolean loggedIn, String url, String email) {
+  public CheckLoginResponse(Boolean loggedIn, String url, String email, Boolean access) {
```

Comment:
Curious, why did you use the boolean object in all these cases?

Reply:
as opposed to the primitive boolean? haha no reason- i just realized I forgot about the primitive boolean. I changed it.